docs/make.jl

```diff
@@ -31,13 +30,12 @@ makedocs(
             "Turtle" => "example/turtle.md",
             "Sky Scraper" => "example/skyscraperBasic.md",
             "Game Of Life" => "example/GameOfLife.md",
-            "Platform 9¾" => "example/platform9_75.md"
+            "Platform 9¾" => "example/platform9_75.md",
             "Whac-a-Block" => "whacABlock.md"
        ],
        "Block Reference" => "blocks.md",
        "API Reference" => "api.md"
-   ],
-   assets = ["assets/custom.css"]
```

Comment:
we don't need a custom css any more?

Reply:
I dislike the current look, appears quite dated and non-modern to me. I may touch it up with a bit of minecrafty theme later.